engine/api/src/main/scala/pl/touk/nussknacker/engine/api/exception/BlacklistedPatternInvocationException.scala

```diff
@@ -0,0 +1,7 @@
+package pl.touk.nussknacker.engine.api.exception
+
+import java.time.Instant
+
+case class BlacklistedPatternInvocationException(message: String,
+                                                 timestamp: Instant = Instant.now(), cause: Throwable = null)
```

Comment:
You do not pass cause parameter to BlacklistedPatternInvocationException - probably can be just removed

Reply:
you are right, thanks